packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -193,17 +193,18 @@ class DocumentCollection {
   }
 
   /**
-   * Migrate an existing unamed index to a named one.
+   * Migrate an existing index to a new name one.
```

Comment:
s/new name one/ new one ?

Reply:
It's true that this sentence was incomprehensible. I detailed it a little more